provisioning/roles/farm/tasks/storage-loop.yml

```diff
@@ -75,5 +75,9 @@
 
   vars:
     mount_path: "{{ (mount == '/')
-                  | ternary(mount, ('', mount) | join('/')) }}"
-    destination_path: "{{ (mount_path, path) | join('/') }}"
+                      | ternary(mount, ('', mount) | join('/')) }}"
```

Comment:
Not particularly sure why this was retabbed.  I assume it's to be consistent with the tabbing on destination_path, but it's inconsistent.  The 'p' in path lines up with the 'm' in mount for destination path, while the | doesn't line up with the m in mount for mount_path.

Reply:
'path' lines up with 'mount' because they are part of the same construct (the input to the 'join').  Actually, one could argue that 'path' should be one character left of where it is because of the () around  'mount'.
As to the preceding '|' it's a subordinate clause continuation of the '(mount...'), not a  an element of the same construct (like 'path').